migrations/20160103164746_set_date_time_fields_with_previous_default_to_null.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class SetDateTimeFieldsWithPreviousDefaultToNull extends AbstractMigration
+{
+    /**
+     * @var array
+     */
+    private $tables = [
+        'talks' => [
+            'created_at',
+            'updated_at',
+        ],
+        'users' => [
+            'activated_at',
+            'created_at',
+            'updated_at',
+            'last_login',
+        ],
+    ];
+
+    public function up()
+    {
+        $this->run('UPDATE `%s` SET `%s` = NULL WHERE `%s` = "1000-01-01 00:00:00"');
```

Comment:
This could be where we have to do branching based on version of MySQL unless this is handled by some other convention. This can't really be a "new problem", I just haven't had to deal with it since nothing I own is running a version of MySQL that new.